### DISCUSSION TITLE
Documentation Clarity

### DISCUSSION THREAD

**Discussion by Rusty-sk** [quasarframework/app-extension-apollo#114] [Q&A] [answered]
I am an older dog getting slower at learning new tricks and I have been lagging to get up to speed on Vue3 implementation in Quasar.

Now with the composition api and the new ways Vue and Quasar deliver a great product my mind is swimming and I am stuck when it says --
![image](https://user-images.githubusercontent.com/35604274/142215840-f32fe81d-3939-404b-a01f-0e50fc30e10a.png)

Formerly it was a matter of entering `vue-apollo` in the boot section but I get a sense you now are supposed to reference the file. An example line of code for the boot section for those of us that are less experienced would be great.

**Comment by Rusty-sk** [comment, marked as answer]
I solved my issue after scouring my project files. Previous versions of GRAPHQL had been injected into the boot section with the `vue-apollo` and the newest version for VUE 3 support it is now just `apollo`.

I thank the crew for developing this extension but the whole premise of frameworks is to spend less time setting up projects and reading various sources of ambiguous documentation & config files and more time developing.

**Reply by smolinari** [reply]
It's open source. If you feel the docs could be better, PRs are welcome. 😁 

Scott
